### DISCUSSION TITLE
Secure-stack Octernship results

### DISCUSSION THREAD

**Discussion by saaxena** [community/community#53227] [GitHub Education]
Hi folks , can anyone tell me when the results of secure-stack Octernship are going to announced, more than 2 weeks have passed after we submitted our assignment.  I had tried to contact our mentors but they didn't replied.
If anyone knows about results kindly tell us .
Thank you

**Comment by Tribhuwan-Joshi** [comment]
If the mentors have sent the list of  selected octerns to GitHub then you will receive an email from GitHub very soon
